src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -672,76 +682,43 @@ export class StudyViewPageStore {
+        await:()=>[this.clinicalAttributes, this.samples],
+        invoke: async ()=>{
+            const filter: ClinicalDataMultiStudyFilter = {
+                attributeIds: [MUTATION_COUNT, FRACTION_GENOME_ALTERED],
+                identifiers: _.map(this.samples.result!, obj => {
+                    return {
+                        "entityId": obj.sampleId,
+                        "studyId": obj.studyId
                     }
                 })
-            ));
-        }
-    });
+            };
 
-    readonly fractionGenomeAltered = remoteData<FractionGenomeAltered[]>({
-        await:()=>[
-            this.samples
-        ],
-        invoke:async()=>{
-            const studyToSamples = _.groupBy(this.samples.result!, s=>s.studyId);
-            return _.flatten(await Promise.all(
-                _.map(studyToSamples, (samples, studyId)=>{
-                    if (samples && samples.length) {
-                        return internalClient.fetchFractionGenomeAlteredUsingPOST({
-                            studyId,
-                            fractionGenomeAlteredFilter: {
-                                sampleIds: samples.map(s=>s.sampleId)
-                            } as FractionGenomeAlteredFilter
-                        });
-                    } else {
-                        return Promise.resolve([]);
-                    }
-                })
-            ));
-        }
-    });
+            let data:ClinicalData[] = await defaultClient.fetchClinicalDataUsingPOST({
+                clinicalDataType: "SAMPLE",
+                clinicalDataMultiStudyFilter: filter
+            });
 
-    readonly mutationCountVsFractionGenomeAlteredData = remoteData({
-        await:()=>[
-            this.mutationCounts,
-            this.fractionGenomeAltered
-        ],
-        invoke: ()=>{
-            const sampleToMutationCount = _.keyBy(this.mutationCounts.result!, c=>c.uniqueSampleKey);
-            const sampleToFga = _.keyBy(this.fractionGenomeAltered.result!, f=>f.uniqueSampleKey);
-            const data = [];
-            for (const sampleKey of Object.keys(sampleToMutationCount)) {
-                const mutationCount = sampleToMutationCount[sampleKey];
-                const fga = sampleToFga[sampleKey];
-                if (mutationCount && fga) {
-                    data.push({
-                        x: fga.value,
-                        y: mutationCount.mutationCount,
-                        studyId: mutationCount.studyId,
-                        sampleId: mutationCount.sampleId,
-                        patientId: mutationCount.patientId,
-                        uniqueSampleKey: mutationCount.uniqueSampleKey
-                    });
+            return _.reduce(_.groupBy(data, datum => datum.uniqueSampleKey), (acc, data) => {
+                if (data.length == 2) { // 2 => number of attribute ids
+                    let _datum: { [id: string]: string | number } = {
```

Comment:
its not a map so I think the type should be more specific than index type, for readability. ie lets declare a type up top `type MutationCountVsFGADatum = { studyId:string, ....}`